applications/nrf_desktop/src/modules/hids.c

```diff
@@ -298,13 +298,13 @@ static int module_init(void)
 	hids_init_param.outp_rep_group_init.cnt = or_pos;
 
 	/* Boot protocol setup */
-	if (IS_ENABLED(CONFIG_DESKTOP_HID_MOUSE)) {
+	if (IS_ENABLED(CONFIG_DESKTOP_HID_REPORT_MOUSE_SUPPORT)) {
 		hids_init_param.is_mouse = true;
 		hids_init_param.boot_mouse_notif_handler =
 			boot_mouse_notif_handler;
 	}
 
-	if (IS_ENABLED(CONFIG_DESKTOP_HID_KEYBOARD)) {
+	if (IS_ENABLED(CONFIG_DESKTOP_HID_REPORT_KEYBOARD_SUPPORT)) {
```

Comment:
Can dongle have both boot reports? (dongle supports HID reports both for mouse and keyboard)